### PR TITLE
n-triples doc patch, refs #92

### PR DESCRIPTION
**Pull request by Sozialarchiv**

### CODE REVIEW

File: packages/formats/README.md [hunk 1]
```diff
@@ -61,6 +61,30 @@ console.log(await getStream(stream))
 
 ```
 
+To get an output in n-triples, n-quads or n3 format the mime-type must be specified:
```
Comment (tpluscode) [inline packages/formats/README.md:64]:
```suggestion
To get an output in n-triples, n-quads or n3 format the sink map interface can be used where media type must be specified:
```

**Review by tpluscode — CHANGES_REQUESTED** [review]
I only propose to rephrase slightly

File: packages/formats/README.md (hunk 1 above)
Comment (Sozialarchiv) [reply packages/formats/README.md:64]: Perfect. Thank you very much.

**Review by tpluscode — APPROVED** [review]

**Comment by tpluscode** [comment]
Thank you @Sozialarchiv

**Pull request merged**